### PR TITLE
USHIFT-2218: Dockerfile for MicroShift

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

File: Dockerfile.microshift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  /usr/src/multus-cni/bin/multus \
+  /usr/src/multus-cni/bin
+ADD ./images/entrypoint.sh /
+
+LABEL io.k8s.display-name="Multus CNI for MicroShift" \
+      io.k8s.description="This is an optional component of MicroShift and provides a meta CNI plugin." \
+      io.openshift.tags="microshift" \
+      maintainer="Patryk Matuszak <pmatusza@redhat.com>"
+
+ENTRYPOINT ["/entrypoint.sh"]
```
Comment (s1061123) [inline Dockerfile.microshift:24]: If you are going to use 'thin_entrypont', you may need to change that (you seems to do 'copy & paste' from current Dockerfile).

Comment (pmtk) [reply Dockerfile.microshift]: It was overridden in the yamls anyway, but you're right - it's unnecessary so I removed it. Thanks!

**Comment by pliurh** [comment]
It looks good to me.

File: Dockerfile.microshift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ENV VERSION=rhel9 COMMIT=unset
+RUN ./hack/build-go.sh
+WORKDIR /
+
+FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
+RUN mkdir -p /usr/src/multus-cni/bin
+COPY --from=rhel9 \
+  /usr/src/multus-cni/bin/thin_entrypoint \
+  /usr/src/multus-cni/bin/multus \
+  /usr/src/multus-cni/bin
```
Comment (s1061123) [inline Dockerfile.microshift:16]: We may just do that. Could you please double check that? (of course, we need COPY, but no need to specify multus + thin_entrypoint. just specify bin path).

````
COPY --from=rhel9 /usr/src/multus-cni/bin /usr/src/multus-cni/bin
```

Comment (pmtk) [reply Dockerfile.microshift:16]: But then it copies also binaries for thick plugin architecture (around +200MB)

Comment (s1061123) [reply Dockerfile.microshift:16]: Oh, I got it. Thank you for the clarification!

**Pull request merged**